### PR TITLE
[Fix] Update Selling Price With Skills

### PR DESCRIPTION
**Pull request by Z1kou**
# Description

The green thumb effect was giving 10%price increase instead of 5% stated in the game, also tried fixing the popup modul of selling when you have the skill, you will still get the updated amount but the pop over still showed the old value of before the skill

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
Yarn dev
Yarn dev Sunflower with skill
NG)

Yarn test 
Yarn test Skill

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/features/crops/components/Plants.tsx [hunk 1]
```diff
@@ -33,8 +33,16 @@ export const Plants: React.FC = () => {
     gameService.send("item.sell", {
       item: selected.name,
       amount,
-    });
-    setToast({ content: "SFL +$" + selected.sellPrice.mul(amount).toString() });
+    })
+      
+    if (inventory["Green Thumb"]?.greaterThanOrEqualTo(1))
```
Comment (j923sneaks) [inline src/features/crops/components/Plants.tsx:38]: why not use `getSellPrice()` from `pricing.ts` instead?

```js
setToast({ content: "SFL +$" + (getSellPrice(selected.name, inventory).mul(amount)).toString() });
```

Comment (Z1kou) [reply src/features/crops/components/Plants.tsx:38]: Spent alot of time trying to set it up with that, errors kept popping left and right so i came up with the alternative that I used here !

**Comment by Z1kou** [comment]
After playing around with the variables for a while trying to find a format that is functioning,consistent with the code and the future code, and without errors I got this! Huge thanks to @j923sneaks  for.... Oh wait You updated the comment with the displaySellPrice() format, OH MY GOD I SPENT SO LONG TRYING TO FIX  THE ERRORS AND IT WAS RIGHT HERE 😂 ,Well this was a great learning experience got to learn more about functions with trial and error and research, @j923sneaks  Thank you bro again ❤️

File: src/features/game/lib/pricing.ts [hunk 2]
```diff
@@ -7,8 +7,8 @@ import { Crop } from "../types/crops";
 export const getSellPrice = (crop: Crop, inventory: Inventory) => {
   let price = crop.sellPrice;
 
-  if (inventory["Green Thumb"]?.greaterThanOrEqualTo(1)) {
-    price = price.mul(1.1);
+  if (inventory["Sunflower Seed"]?.greaterThanOrEqualTo(1)) {
```
Comment (j923sneaks) [inline src/features/game/lib/pricing.ts:10]: I think we should retain `Green Thumb` here :)

File: src/features/crops/components/Plants.tsx [hunk 3]
```diff
@@ -29,12 +29,17 @@ export const Plants: React.FC = () => {
...
+
+
   const sell = (amount = 1) => {
     gameService.send("item.sell", {
       item: selected.name,
       amount,
-    });
-    setToast({ content: "SFL +$" + selected.sellPrice.mul(amount).toString() });
+    })
+      {
```
Comment (j923sneaks) [inline src/features/crops/components/Plants.tsx:39]: you can omit the square brackets

**Comment by j923sneaks** [comment]
also @adamhannigan some were raising that the 10% bug persists when saving? shouldn't it be also handled in the backend?

**Comment by jcads** [comment]
> After playing around with the variables for a while trying to find a format that is functioning,consistent with the code and the future code,

@Z1kou you can run `yarn format` to run the linter and format the code.

File: src/features/crops/components/Plants.tsx (hunk 3 above)
Comment (Z1kou) [reply src/features/crops/components/Plants.tsx:39]: It is because there is only one command in them right ?

**Review by adamhannigan — APPROVED** [review]
LGTM. Nice work again! 💪

**Pull request merged**